gap/main.gi

```diff
@@ -43,6 +44,17 @@ InstallGlobalFunction( DoEchelonMatTransformationBlockwise,
             fi;
         fi;
 
+        # verify only makes sense if withTrafo is true
+        if "verify" in recnames then
+            verify := options.verify;
+        else
+            verify := withTrafo;
+        fi;
+        if verify and not withTrafo then
```

Comment:
Even if we would adapt this in `Chief` (see my other comment) we could still throw this error message because `Chief` wouldn't do what the user expected it to do. But I think that it would be good to make `Chief` as error-proof as possible, wouldn't it?

Reply:
I think in principle you're right, but hopefully we can get rid of `Chief` soon (after #170). If we were to add another check to `Chief` this would be duplicate to the check in `DoEchelon..`, once these two functions are merged.